.circleci/config.yml

```diff
@@ -208,14 +199,13 @@ jobs:
           paths:
             - .yarn/unplugged
             - node_modules/.cache
-      - run: tar -czf build.tgz .gcloudignore app.yaml build config
```

Comment:
`app.yaml` here and in the `persist_to_workspace` step below were not part of #3004.

`app.yaml` has been in these lists for years (181bdb27dbc50ddb951859feb9682a3d559e8322).

Reply:
oh, that's not good. fixed now.